src/tests/run_tests.sh

```diff
@@ -11,10 +11,13 @@ tot=0
 QL_IMAGE=${QL_PREV_IMAGE} docker-compose -f ../../docker/docker-compose-dev.yml up -d
 sleep 10
 
+ORION_HOST=`docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' docker_orion_1`
```

Comment:
I wonder if you didn't need to do this also for the tests in the reporter package. Note that in QL not all the tests are using the `docker/docker-compose-dev/yml` file (where you are giving a name to these containers).

Reply:
The "container_name" variable was used to fetch the IP address for inter container communication. Now we are updating the script file for getting the IP address dynamically by based on the port on which Orion and Quantumleap are running.

Reply:
yep, I agree we should check all the other test cases too, though I'd have expected them to fail...

Reply:
sure, I understood why the naming was needed, I was just curious to see it wasn't needed in at least the docker-compose used in the reporter tests. Anyway, nevermind.